### ISSUE TITLE
Predicting future values  - from all training data

### ISSUE THREAD

**Issue by andrewcztrack** [cvxgrp/low_rank_forecasting_code#1]
Hi @sbarratt !!!

I hope your well! 
Love the paper!! was wondering how do we use the algorithm to predict future values.
ie, not creating a test set, use all the data as the training set and then get a vector of future values.
For example if the training set is 150 points, how do i get the forecast for points 151-160?

Thankyou!
Best,
Andrew

**Comment by andrewcztrack** [comment]
Hi @sbarratt !

I hope your well.

I have had a deep look at the code and it looks like your using the test set to implement the forecast?

Is this correct?

Kind regards and thanks,
Andrew

**Comment by sbarratt** [comment]
Andrew,

You have two options:

You could manually construct the p_t vector by stacking previous values of x_t.

You could call the fit method with data you want to forecast using zero fitting iterations, and it returns P and F. (You would do P * theta) to predict. (This is what I do for most examples for the test set.)
